ivy/functional/frontends/paddle/tensor/linalg.py

```diff
@@ -144,3 +144,9 @@ def bmm(x, y, transpose_x=False, transpose_y=False, name=None):
 @to_ivy_arrays_and_back
 def matrix_power(x, n, name=None):
     return ivy.matrix_power(x, n)
+
+
+@with_unsupported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
```

Comment:
This should be specified as supported 😅
Using `with_supported_dtypes` decorator

Reply:
> This should be specified as supported 😅 Using `with_supported_dtypes` decorator

i dont know why am over looking this line

Reply:
This decorator is for filtering unsupported dtypes of that function and from the docs this function supports only `float32` and `float64`